scripts/mock.gen.sh

```diff
@@ -43,10 +42,10 @@ do
   package_name=$(basename "$(dirname "$output_path")")
   echo "Generating ${output_path}..."
   mockgen -package="${package_name}" -destination="${output_path}" "${src_import_path}" "${interface_name}"
-
-  go-license \
-  --config=./license.yml \
-  "${output_path}"
 done < "$input"
 
+# add license headers to mock files
+# ignore errors since mockgen files give warnings when edited
+hawkeye format || true
```

Comment:
not a fan of this, but otherwise we need go-license and using two license linters seems worse